website/content/en/docs/building-operators/ansible/reference/advanced_options.md

```diff
@@ -263,7 +263,29 @@ Using the `--ansible-log-events` CLI flag, you can determine to what degree the
 2. `Tasks` - Only Ansible Tasks will be outputted.
 3. `Everything` - All info logs and all tasks will be outputted.
 
-If you want more control over the logs that are outputted, consider using the [Zap Logger][Zap-Logger] and [verbosity annotations][verbosity-annotations] in tandem with the `--ansible-log-events` CLI flag. 
+If you want more control over the logs that are outputted, consider using the [Zap Logger][Zap-Logger] and [verbosity annotations][verbosity-annotations] in tandem with the `--ansible-log-events` CLI flag.
+
+## `ansible.sdk.operatorframework.io/reconcile-period` Custom Resource Annoation
+
+While running a Ansible-based operator, the reconcile-period can be specified through the custom resource's annotations under the `ansible.sdk.operatorframework.io/reconcile-period` key. 
```

Comment:
While running an Ansible-based operator, the reconcile-period can be specified through the custom resource's annotations under the `ansible.sdk.operatorframework.io/reconcile-period` key.